File: internal/provider/dependence/neo4jcsv/testdata/src/golang/w.go
package w

type S struct {
	F   int
	Arr [4]int
	M   map[string]int
}

var G int

func run(s *S, p *int, i int) {
	x := 1
	x += 2
	x++
	s.F = 3
	s.Arr[i] = 4
	*p = 5
	*p += 1
	G = x
	a, b := x, x
	a, b = b, a
	s.M["k"] = 6
	_ = a
	_ = b
}

File: internal/provider/dependence/neo4jcsv/testdata/src/c/w.c
struct S { int f; int arr[4]; };
int g;
void run(struct S *s, int *p, int i) {
  int x = 1; x += 2; x++; --x;
  s->f = 3; s->arr[i] = 4; (*p) = 5; *p += 1; g = x;
  int y = x = 7;
}
